src/rdb.c

```diff
@@ -2112,8 +2112,8 @@ robj *rdbLoadObject(int rdbtype, rio *rdb, sds key, int dbid) {
                     decrRefCount(o);
                     return NULL;
                 }
-                streamConsumer *consumer =
-                    streamLookupConsumer(cgroup,cname,SLC_NONE,NULL);
+                streamConsumer *consumer = streamCreateConsumer(cgroup,cname,
+                                                                NULL,0);
```

Comment:
i'd vote to make this a one liner.. we don't have a strict line length limit, and a wraparound for this seems less readable for me.
```suggestion
                streamConsumer *consumer = streamCreateConsumer(cgroup,cname,NULL,0);
```

Reply:
Okay, got it
> we don't have a strict line length limit